ui/component/viewers/videoViewer/view.jsx

```diff
@@ -535,6 +532,7 @@ function VideoViewer(props: Props) {
         activeLivestreamForChannel={activeLivestreamForChannel}
         defaultQuality={defaultQuality}
         doToast={doToast}
+        claim={claim}
```

Comment:
Still doesn't work.  EEVBlog chapters appearing in a livestream.

<img width="580" alt="image" src="https://user-images.githubusercontent.com/64950861/175968867-2a701a0f-819a-49e8-baa5-6d6e3d8a0e01.png">

Side note that if going to pass in `claim`, there will be a few redundant props to remove.
The reason the claim data was split is to minimize prop diff